### PR TITLE
Added mock in prices page

### PR DESCRIPTION
**Pull request by MaxOvs19**

### CODE REVIEW

**Comment by MaxOvs19** [comment]
[Add mocks for price page](https://trello.com/c/kaIKRTFF/31-add-mocks-for-price-page)

File: src/app/prices/interfaces/contribution.prices.interface.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface Icontribution {
```
Comment (alexey-ovsyanikov-noveo) [inline src/app/prices/interfaces/contribution.prices.interface.ts:1]: IContribution

File: src/app/prices/interfaces/prices.interface.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export interface Iprice {
```
Comment (alexey-ovsyanikov-noveo) [inline src/app/prices/interfaces/prices.interface.ts:1]: @MaxOvs19  IPrice

File: src/app/prices/prices.component.html [hunk 3]
```diff
@@ -11,80 +11,49 @@ <h4>Здесь Вы можете ознакомиться с ценами на 
...
-    <div class="price-list">
-      <div class="price-list__title">
-        <h2>Полный день в саду</h2>
-      </div>
-
-      <div class="price-list__body">
-        <div class="price-list__age">
-          <h5>Возраст:</h5>
-          <p>от 1,5 до 5,5 лет</p>
+    <ng-container *ngFor="let description of price">
```
Comment (alexey-ovsyanikov-noveo) [inline src/app/prices/prices.component.html:14]: @MaxOvs19 need to replace "price" on "prices".
"description" -> item as well

File: src/app/prices/prices.component.html [hunk 4]
```diff
@@ -11,80 +11,49 @@ <h4>Здесь Вы можете ознакомиться с ценами на 
...
+        <div class="price-list__body">
+          <div class="price-list__age">
+            <h5>Возраст:</h5>
+            <p>{{ description.age }}</p>
+          </div>
+
+          <div class="price-list__description">
+            <h5>В стоимость входит:</h5>
+
+            <p>{{ description.includedPrice[0] }}</p>
```
Comment (alexey-ovsyanikov-noveo) [inline src/app/prices/prices.component.html:29]: We can use *ngFor for it. @MaxOvs19

File: src/app/prices/prices.component.ts [hunk 5]
```diff
@@ -4,13 +4,59 @@ import { Router } from '@angular/router';
...
+      includedPrice: [
+        '5-ти разовое питание ',
+        'Игровая деятельность',
+        'Прогулки',
+        'Дневной сон ',
+        'Групповые игры на английском языке',
+        'Групповые логопедические игры',
+      ],
+      time: 'C 8-00 до 19-00',
+      price: '25 000 ₽',
```
Comment (alexey-ovsyanikov-noveo) [inline src/app/prices/prices.component.ts:30]: Please replace string on number for price field and use number pipe. It's builtin Angular pipe

File: src/app/prices/prices.component.ts [hunk 6]
```diff
@@ -4,13 +4,59 @@ import { Router } from '@angular/router';
...
+      age: 'от 1,5 до 5,5 лет',
+      includedPrice: [
+        '5-ти разовое питание ',
+        'Игровая деятельность',
+        'Прогулки',
+        'Дневной сон ',
+        'Групповые игры на английском языке',
+        'Групповые логопедические игры',
+      ],
+      time: 'C 8-00 до 19-00',
```
Comment (alexey-ovsyanikov-noveo) [inline src/app/prices/prices.component.ts:29]: @MaxOvs19 we need to separate time field. I propose to use time: { from: '8', to: '16:30' } for example

**Review by alexey-ovsyanikov-noveo — APPROVED** [review]

**Pull request merged**